src/main/resources/static/css/styles.css

```diff
@@ -51,3 +51,42 @@ main{
 hr.style1{
 	border-top: 1px solid #8c8b8b;
 }
+
```

Comment:
I think we can delete code from line 55 since we have it in the splitscreen.css file

Reply:
Yes, you're right!